<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="displayListWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>161</y>
      <width>256</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>10</y>
      <width>261</width>
      <height>146</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="clickTimeButton">
       <property name="text">
        <string>現在時間是？</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clickTestButton">
       <property name="text">
        <string>嗨起來</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clickDateButton">
       <property name="text">
        <string>今天日期是?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectRadionButton">
       <property name="text">
        <string>你最後選擇的飲料是？</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="clickWaterRadioButton">
       <property name="text">
        <string>水</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="clickJuiceRadioButton">
       <property name="text">
        <string>果汁</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="clickColaRadioButton">
       <property name="text">
        <string>可樂</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
